src/averageAges.js

```diff
@@ -1,25 +1,30 @@
 'use strict';
 
+// const { map } = require("./people");
+
 /**
  * Implement calculateMenAverageAge function
  *
  * Function returns average age of men in array. If `century` is specified then
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of person's death by 100:
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
```

Comment:
```suggestion
  const men =  people.filter( person => century
    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
    : person.sex === 'm');
```

Reply:
I think, your variant is redundant. Try to make it in shorter view.

Reply:
Agree, this is shorter and more elegant solution.